src/js/personalization/dashboard-beta/components/FormItem.jsx

```diff
@@ -10,15 +10,15 @@ class FormItem extends React.Component {
     const formId = savedFormData.form;
     const { last_updated: lastSaved, expires_at: expirationTime } = savedFormData.metadata;
     const lastSavedDateTime = moment.unix(lastSaved).format('M/D/YYYY [at] h:mm a');
-    const expirationDate = moment.unix(expirationTime).format('M/D/YYYY');
-    const isExpired = moment(expirationDate).isBefore();
+    const expirationDate = moment.unix(expirationTime).format('MMM DD');
+    const isExpired = moment.unix(expirationTime).isBefore();
     const activeView = (
```

Comment:
Yes, needed to add this because the date was being parsed incorrectly. Apparently the Unix version of the date from `expirationDate` above was not carrying over when parsing the time for the `isExpired` variable. You don't need to add Unix unless the time being parsed is a unix time